spec/models/banner_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe Banner, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
```

Comment:
Since the model is so simple, maybe just add a `it { should validate_presence_of :message }` (after adding the validation in the model above)

Reply:
Done.